File: frameworks/dotnet/aspnet-mvc/Implementation/Controllers/BaselineController.cs
using Microsoft.AspNetCore.Mvc;

namespace Implementation.Controllers;

/// <summary>baseline: the dispatch floor, with nothing serialised.</summary>
[ApiController]
public sealed class BaselineController : ControllerBase
{
    [HttpGet("/plaintext")]
    public ContentResult Plaintext() => Content("Hello, World!", "text/plain");
}

File: frameworks/node/nestjs/Implementation/baseline/baseline.controller.ts
import { Controller, Get, Header } from '@nestjs/common';

/** baseline: the dispatch floor, with nothing serialised. */
@Controller()
export class BaselineController {
  // rb:handler baseline.plaintext
  @Get('plaintext')
  @Header('Content-Type', 'text/plain; charset=utf-8')
  plaintext(): string {
    return 'Hello, World!';
  }
}
